docker/complement/Dockerfile

```diff
@@ -1,22 +1,40 @@
-# A dockerfile which builds an image suitable for testing Synapse under
-# complement.
+# This dockerfile builds on top of 'docker/Dockerfile-worker' in matrix-org/synapse
+# by including a built-in postgres instance, as well as setting up the homeserver so
+# that it is ready for testing via Complement.
+#
+# Instructions for building this image from those it depends on is detailed in this guide:
+# https://github.com/matrix-org/synapse/blob/develop/docker/README-testing.md#testing-with-postgresql-and-single-or-multi-process-synapse
+FROM matrixdotorg/synapse-workers
```

Comment:
sure it does, at the expense of building the entire Synapse image from scratch for each version ;)

Reply:
Ah, you're running it with the pre-builts? Yeah, that sounds like it'd be good to support

Reply:
I will note just in case you think it would be useful: we could revive #11852 to publish the synapse-worker images to GHCR if that would make this kind of investigation easier.